core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/quota/UserQuotaResource.scala

```diff
@@ -227,6 +284,20 @@ class UserQuotaResource {
     getUserCreatedFile(current_user.getUid)
   }
 
+  @GET
+  @Path("/dataset_size")
+  @Produces(Array(MediaType.APPLICATION_JSON))
+  def getDatasetSize(@Auth current_user: SessionUser): Long = {
+    getUserDatasetSize(current_user.getUid)
+  }
+
+  @GET
+  @Path("/uploaded_dataset")
+  @Produces(Array(MediaType.APPLICATION_JSON))
```

Comment:
please change this endpoint function. Name the path as `/"number_of_datasets"`.  Let this endpoint simply return the number instead of an array of datasets.